Code/Framework/AzToolsFramework/AzToolsFramework/UI/Logging/LogLine.cpp

```diff
@@ -382,7 +382,7 @@ namespace AzToolsFramework
             using namespace AZ::Debug;
 
             const AZStd::string separator = " | ";
```

Comment:
minor: the case or Origin is inconsistent with separator, severity, m_window, m_message. I'd recommend making it `origin`